tutorials/SpectroscopicDataReductionBasics/2-WavelengthCalibration.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>because each lamp is at a different pressure </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>